py3status/modules/file_status.py

```diff
@@ -68,17 +111,40 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
```

Comment:
We should move this to `post_config_hook` too. See `hamster` for an example.

Reply:
`hamster` from which branch? :)

Reply:
Master Splinter! Or see `weather_yahoo`.. or to be more specific, do like this...
```
STRING_NO_PATH = 'missing path'
```
```
    def post_config_hook(self):
        if not self.py3.check_commands(self.path):
            raise Exception(STRING_NO_PATH)
```

Reply:
Ha ok I was far away